doc/_sphinx/make.bat

```diff
@@ -26,11 +27,16 @@ if errorlevel 9009 (
 	exit /b 1
 )
 
-%SPHINXBUILD% -M %1 %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O%
+if "%1" == "livehtml" (

```

Comment:
Is this really how one does make targets on windows?
I would have guessed it is more like with `:help` in the end of the file.

Reply:
Maybe there is no clean way of doing it, but this way feels more like a makefile:
https://superuser.com/a/717465

Reply:
Well you can do it several ways.  If statements or named blocks.

Reply:
Updated for blocks.  Its the same difference.